### PR TITLE
Add/input component

### PR DESCRIPTION
**Pull request by SimonNord**
No tests has been created for this component yet. WIP

### CODE REVIEW

File: src/components/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  font-size: 16px;
+  outline: 0;
+  border: 2px solid #eaeaea;
+`;
+
+const Input = () => {
+  return <StyledInput />;
+};
+
+export default Input;
```
Comment (MoSattler) [inline src/components/Input/Input.tsx:16]: The problem is, that you did not declare any `Props` on `<Input />`. But you want it to have the props a normal `input` tag would have. 
The easiest way to solve this in this situation is to do the following:
```suggestion
const StyledInput = styled.input`
  border-radius: 8px;
  padding: 20px;
  font-size: 16px;
  outline: 0;
  border: 2px solid #eaeaea;
`;

export default StyledInput;
```

Comment (MoSattler) [inline src/components/Input/Input.tsx:16]: The problem is, that you did not declare any `Props` on `<Input />`. But you want it to have the props a normal `input` tag would have. 
The easiest way to solve this in this situation is to do the following:
```suggestion
export default StyledInput;
```

File: src/components/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+const StyledInput = styled.input`
+  border-radius: 8px;
+  padding: 20px;
+  font-size: 16px;
+  outline: 0;
+  border: 2px solid #eaeaea;
```
Comment (MoSattler) [inline src/components/Input/Input.tsx:9]: Is this in our variables file?

File: src/components/Input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+const StyledInput = styled.input`
```
Comment (MoSattler) [inline src/components/Input/Input.tsx:4]:
```suggestion
const Input = styled.input`
```
I would also rename this to `Input` then.

Comment (MoSattler) [inline src/components/Input/Input.tsx:4]:
```suggestion
const Input = styled.input`
```

File: src/components/Input/Input.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  padding: 20px;
+  font-size: 16px;
+  outline: 0;
+  border: 2px solid #eaeaea;
+`;
+
+const Input = () => {
+  return <StyledInput />;
+};
+
```
Comment (MoSattler) [inline src/components/Input/Input.tsx:15]: The problem is, that you did not declare any `Props` on `<Input />`. But you want it to have the props a normal `input` tag would have. 
The easiest way to solve this in this situation is to do the following:

```suggestion
```

this way you are directly exporting the input directly, without another wrapper. 

There are also ways to just pass down props (e.g. in the scenario where you want all the `input` props *plus* some custom ones, but that might be another lesson for the future).

Comment (MoSattler) [inline src/components/Input/Input.tsx:15]: The problem is, that you did not declare any `Props` on `<Input />`. But you want it to have the props a normal `input` tag would have. 
The easiest way to solve this in this situation is to do the following:


```suggestion
const Input = styled.input`
  border-radius: 8px;
  padding: 20px;
  font-size: 16px;
  outline: 0;
  border: 2px solid #eaeaea;
`;

```

this way you are directly exporting the input directly, without another wrapper. 

There are also ways to just pass down props (e.g. in the scenario where you want all the `input` props *plus* some custom ones, but that might be another lesson for the future).

Comment (SimonNord) [reply src/components/Input/Input.tsx:15]: Ah okey, thanks, that what we talked about before

File: src/components/Input/Input.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  alignItems: "center",
+  justifyContent: "center",
+  width: "100%",
+  height: "100%",
+  background: "white",
+};
+
+export const InputStory = () => (
+  <div style={StorybookWhiteCentered}>
+    <Input type="text" placeholder="Name"></Input>
```
Comment (MoSattler) [inline src/components/Input/Input.stories.tsx:21]: Optional suggestion, but you can shorten this to
```suggestion
    <Input type="text" placeholder="Name" />
```

File: src/components/Input/Input.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  title: "Input",
+};
+
+const StorybookWhiteCentered = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  width: "100%",
+  height: "100%",
+  background: "white",
```
Comment (MoSattler) [inline src/components/Input/Input.stories.tsx:16]: you could also set the background like [here](https://github.com/HandsOnLearning/the-project/blob/master/src/components/FormattedText/FormattedText.stories.tsx#L7-L14).
